.travis.yml

```diff
@@ -7,56 +7,16 @@ services:
 matrix:
   include:
     - os: linux
-      name: "Analyzer test cases"
+      name: "Analyzer and web test cases"
```

Comment:
Please add the same name for linux too `"Analyzer and web test cases (sqlite, psycopg2, pg8000)"`